sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -397,7 +397,7 @@ public boolean isValid() {
     private Object describeType(Type type) {
```

Comment:
According to the bytebuddy javadoc of that occurrence, visitLocalVariable actually takes the descriptor (_desc - the type descriptor of this local variable._), while visitFrame takes internal names (_Reference types are represented by String objects (representing internal names)_). Anyway further occurrences should be caught by https://github.com/apache/incubator-beam/pull/954